inventory/README.md

```diff
@@ -0,0 +1,62 @@
+# Inventory component
+Inventory component for screencrash, with support for items and achievements. Custom made specifically for the show Apornas ö.
+Run component to handle backend calls and then add a web view with the `media`-component to visualize the result, or visit webpage at http://localhost:4218.
+Manual changes can be done through screencrash commands or via the update subpage (http://localhost:4218/update).
+
+Available items and achivements can be set up with configuration files. See more in [Required resources](#required-resources)
+
+## Requirements
+NodeJS v10.0+ . Make is nice but not required.
+
+## Required resources
+To customize the inventory to your liking a number of resources (mainly images) are needed. The root folder for all of these files are `public/inventory-data`:
```

Comment:
Do I understand it correctly that this should be zipped and stored in Core's resource dir? It would be nice if at least inventory-data.json could live outside a zip file to make it easier to edit, but that could be a future thing.

Reply:
Will make ticket!